__init__.py

```diff
@@ -191,7 +192,10 @@ def _start_new_timer(self, message):
         else:
             timer = self._build_timer(duration, name)
             self.active_timers.append(timer)
-            self.active_timers.sort(key=lambda x: x.expiration)
+            self.active_timers.sort(key=lambda tmr: tmr.expiration)
+            if len(self.active_timers) == 1:
+                self._show_gui()
+                self._start_display_update()
             self._speak_new_timer(timer)
             self._save_timers()
 
```

Comment:
In cancel_timers() this should come from a voc file or it will only work for english. 
or message.data.get("all")

Reply:
If you look at the other side of the `or` condition, you will see that `self.all_timers_words` is being checked.  If you look in the constructor, you will see that it is using a vocabulary file.  If `all` is in the message, it simply means that the intent parser caught it.